### DISCUSSION TITLE
What is the support for Treat looks like

### DISCUSSION THREAD

**Discussion by zainul-franciscus-myob** [vanilla-extract-css/vanilla-extract#27] [Q&A] [answered]
Hi,

What is the support of Treat looks like since vanilla-extract is released ? 

Will Treat still be maintained by the team who created by Treat ? For example: Will there still feature development and fixes put into Treat since now vanilla-extract is released ?

Thank you

**Comment by mattcompiles** [comment, marked as answer]
Hey 👋 ,

Good question, we should probably add something to the treat site/README.
There won't be any further feature development on treat as our focus is on vanilla-extract going forward. 

Is migrating to vanilla-extract an issue for your project? Or is there something treat does that you can't do with vanilla?

**Reply by zainul-franciscus-myob** [reply]
Hi, 

Our team is considering what are the pro and cons of moving to vanilla extract. Knowing that new features will be put into vanilla extract may sway everyone's decision to move to vanilla.

Thank you

**Reply by zainul-franciscus-myob** [reply]
@markdalgleish @mattcompiles Am I right to say that there won't be anymore bug fixes in Treat code base since now vanilla-extract has been published ?

**Reply by markdalgleish** [reply]
I'd say it's unlikely, depending on the severity of the issue.

**Reply by markdalgleish** [reply]
We'll still be running treat in production for a while, even as we're rolling out vanilla-extract, but it's been pretty solid for us.

**Reply by zainul-franciscus-myob** [reply]
@markdalgleish @mattcompiles when will you guys be releasing a major non-alpha version? We would like to re-validate our projects when that happens :)
